UpdateService/handler.ts

```diff
@@ -137,10 +140,22 @@ const updateServiceTask = (
           ...retrievedService,
           ...servicePayload,
           service_id: serviceId,
-          service_metadata: {
-            ...servicePayload.service_metadata,
-            token_name: adb2cTokenName
-          }
+          // Only Admins can change service category and custom_special_flow name
```

Comment:
It's not clear to me how "being admin" is ensured in this flow

Reply:
Good point. I've tried to extend the comment in f2482cb